Sources/ListLabel25.pas

```diff
@@ -260,20 +267,34 @@   TListLabel25 = class(TComponent, ILlDomParent)
   end;
 
   //Core class
-   LlCore = class (TObject)
-   private
+  LlCore = class (TObject)
+  private
     fParentObject: TListLabel25;
-   public
-   Constructor Create(ParentObject: TListLabel25);
-   function LlGetOptionString(OptionIndex: integer; var Value: TString): integer;
-   function LlDefineVariableExt(FieldName: String; Contents: String; FieldType: integer): integer;
-   function LlDefineVariableExtHandle(FieldName: String; Handle : Cardinal; FieldType: integer): integer;
-   function LlDefineFieldExt(FieldName: String; Contents: String; FieldType: integer): integer;
-   function LlDefineFieldExtHandle(FieldName: String; Handle : Cardinal; FieldType: integer): integer;
-   function LlSetOption(OptionIndex: integer; Value: lParam): integer;
-   function LlGetOption(OptionIndex: integer): integer;
-   function LlSetOptionString(OptionIndex: integer; Value: TString): integer;
-
+  public
+    Constructor Create(ParentObject: TListLabel25);
+    function LlGetOptionString(OptionIndex: integer; var Value: TString): integer;
+    function LlDefineVariableExt(FieldName: String; Contents: String; FieldType: integer): integer;
+    function LlDefineVariableExtHandle(FieldName: String; Handle : Cardinal; FieldType: integer): integer;
+    function LlDefineFieldExt(FieldName: String; Contents: String; FieldType: integer): integer;
+    function LlDefineFieldExtHandle(FieldName: String; Handle : Cardinal; FieldType: integer): integer;
+    function LlSetOption(OptionIndex: integer; Value: lParam): integer;
+    function LlGetOption(OptionIndex: integer): integer;
+    function LlSetOptionString(OptionIndex: integer; Value: TString): integer;
+    function LlLocAddDesingLcId(Language: TLlLanguage): Integer;
+    function LlLocAddDictionaryEntry(Language: TLlLanguage; Key: TString; Value: TString; DictionaryType: TllDictionaryType): Integer;
```

Comment:
Have your thought of adding the actual dictionary class from the "classic" VCL component? That way, other users might be able to reuse existing code 1:1. I'm talking about this class here:

```
TDictionary = class(TObject)
  private
    FFields: TDicFields;
    FTables: TDicData;
    FRelations: TDicData;
    FSortOrders: TDicData;
    FParentObj: TL25_;
    FStaticTexts: TDicData;
    procedure SetRelations(const Value: TDicData);
    procedure SetFields(const Value: TDicFields);
    procedure SetTables(const Value: TDicData);
    procedure SetStaticTexts(const Value: TDicData);
    procedure SetSortOrders(const Value: TDicData);
  public
    property Fields: TDicFields read FFields write SetFields;
    property Tables: TDicData read FTables write SetTables;
    property Relations: TDicData read FRelations write SetRelations;
    property SortOrders: TDicData read FSortOrders write SetSortOrders;
    property StaticTexts: TDicData read FStaticTexts write SetStaticTexts;
    constructor Create(ParentObj: TL25_);
    destructor Destroy; override;
    procedure Clear;
  end;
```

Reply:
Thanks for the hint. I removed the functions. I' going to integrate the Dictionary class from the classic component